Makefile

```
@@ -99,61 +99,41 @@ conventions-check: ## Check Rego policy files for convention violations
 	if [[ -n "$${OUT}" ]]; then echo "$${OUT}"; exit 1; fi
 
 .PHONY: ready
-ready: fmt-amend docs-amend ## Amend current commit with fmt and docs changes
+ready: fmt-amend ## Amend current commit with fmt changes
 
 ##@ Documentation
 
-DOCSRC=./docsrc
-DOCS_TMP_JSON=$(DOCSRC)/annotations-data.json
-ACCEPTABLE_BUNDLES_YAML=$(DATA_DIR)/acceptable_tekton_bundles.yml
+ANTORA_DOCS=./antora/docs
+ANNOTATIONS_JSON=$(ANTORA_DOCS)/rule_annotations.json
 
-# Use gomplate to generate Asciidoc files as input for Antora
-# (For preview and to produce our local Github pages doc only.
-# The official HACBS Antora does its own import and docs build.)
-#
-DOCS_PAGES_DIR=./antora-docs/modules/ROOT/pages
-DOCS_ALL=\
-  $(DOCS_PAGES_DIR)/index.adoc\
-  $(DOCS_PAGES_DIR)/release_policy.adoc\
-  $(DOCS_PAGES_DIR)/pipeline_policy.adoc\
-  $(DOCS_PAGES_DIR)/acceptable_bundles.adoc
-
-$(DOCS_TMP_JSON):
-	@opa inspect --annotations --format json $(POLICY_DIR) > $@
-
-# Beware vim .swp files in the templates directory will break this
-$(DOCS_PAGES_DIR)/%.adoc: $(DOCSRC)/%.adoc.tmpl
-	@gomplate \
-	  --template templates=$(DOCSRC)/templates/ \
-	  --datasource rules=$(DOCS_TMP_JSON) \
-	  --datasource bundles=$(ACCEPTABLE_BUNDLES_YAML) \
-	  --file $< |
-	  cat -s > $@
-
-.PHONY: docs-clean
-docs-clean:
-	@rm -rf $(DOCS_ALL) $(DOCS_TMP_JSON)
-
-.PHONY: docs-build
-docs-build: docs-clean $(DOCS_TMP_JSON) $(DOCS_ALL) ## Generate documentation. Use this before commit if you modified any rules or annotations
-	@rm $(DOCS_TMP_JSON)
+$(ANNOTATIONS_JSON):
+# Use jq sort_by here because otherwise the order is different every time
+	@opa inspect --annotations --format json $(POLICY_DIR) | jq -S '.annotations |= sort_by(.location.file, .location.row)' > $@
 
-.PHONY: docs-refresh
-docs-refresh: docs-build docs-render
+.PHONY: clean-annotations
+clean-annotations:
+	@rm -f $(ANNOTATIONS_JSON)
 
-.PHONY: docs-amend
-docs-amend: docs-build ## Update the docs and amend the current commit
-	@git --no-pager diff $(DOCS_ALL)
-	@echo "Amend commit '$$(git log -n1 --oneline)' with the above diff?"
-	@read -p "Hit enter to continue, Ctrl-C to abort."
-	git add $(DOCS_ALL)
-	git commit --amend --no-edit
+.PHONY: annotations
+annotations: clean-annotations $(ANNOTATIONS_JSON) ## Refresh the rego annotations file
+
+.PHONY: docs-refresh
+docs-refresh: annotations docs-render
 
 # Use Antora to build html from the Asciidoc files under antora-docs
 #
 .PHONY: docs-render
 docs-render: ## Builds the Antora documentation with the local changes
-	@npm exec -y --quiet antora generate --clean --fetch antora-playbook.yml
+	@npm install --no-progress
+	@npm exec -y --quiet -- antora --stacktrace generate --clean --fetch antora-playbook.yml
+
+SHORT_SHA=$(shell git rev-parse HEAD | head -c 7 )
```

No need to pipe to `head`

```suggestion
SHORT_SHA=$(shell git rev-parse --short HEAD)
```